jbpm-console-ng-workbench-integration/jbpm-console-ng-workbench-integration-client/src/main/java/org/jbpm/console/ng/wi/client/editors/deployment/descriptor/DeploymentDescriptorViewImpl.java

```diff
@@ -1286,6 +1288,13 @@ private void setSelectedValue( final ListBox listbox,
         }
     }
 
+    protected void fillRuntimeStrategiesDropdown(ListBox runtimeStrategyDropdown){
+        RuntimeStrategy[] strategies = RuntimeStrategy.values();
+        for(RuntimeStrategy runTimeStrategy : strategies){
+            runtimeStrategyDropdown.addItem(ClientRuntimeStrategy.getRuntimeTranslation(runTimeStrategy), runTimeStrategy.name());
```

Comment:
This could be decoupled, moving the RuntimeStrategy handling part to the Presenter instead, ideally we want the view to not know about the model.